src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+
+    &:last-child {
+      margin: 0;
+    }
+
+    &:nth-child(-n + 4) {
+      background-image: url(../images/star-active.svg);
```

Comment:
The `:nth-child(-n + 4)` selector will always apply the active star image to the first four stars. If the number of active stars should be dynamic based on a rating value, consider using JavaScript to dynamically apply a class or style to the appropriate number of stars.